.github/workflows/check-links.yml

```diff
@@ -66,6 +66,9 @@ jobs:
           --exclude "^https://www.pygmt.org/%7B%7Bpath%7D%7D"
           --exclude "^https://www.researchgate.net/"
           --exclude "^https://zenodo.org/badge/DOI/"
+          --exclude-path "file:///home/runner/work/pygmt/pygmt/repository/doc/gallery/index.rst"
+          --exclude-path "file:///home/runner/work/pygmt/pygmt/repository/doc/tutorials/index.rst"
+          --exclude-path "file:///home/runner/work/pygmt/pygmt/repository/doc/intro/index.rst"
```

Comment:
I don't think this is a good solution, since it contains absolute paths that are specific to CI. Can we just remove the "Start by looking at our Intro, Tutorials, and Gallery." sentence from `install.md`?

Reply:
Aggree regarding not adding absolute paths.
Besides removing this sentence, we can use the full URL?

Reply:
It's not ideal to use full URLs, specially that the full URLs are pointing to the latest version.